src/object.c

```diff
@@ -736,7 +743,7 @@ size_t objectComputeSize(robj *o, size_t sample_size) {
             asize = sizeof(*o)+sizeof(dict)+(sizeof(struct dictEntry*)*dictSlots(d));
             while((de = dictNext(di)) != NULL && samples < sample_size) {
                 ele = dictGetKey(de);
-                elesize += sizeof(struct dictEntry) + sdsAllocSize(ele);
+                elesize += sizeOfStringObject(ele);
```

Comment:
After carefully checking this version's calling of setTypeAdd, I agree with you.
I copied the implementation from this unstable branch to stable version of 3.2.8 and found the bug.
So I fixed it with version 3.2.8 orignally where the ele should be robj * but not sds.
And I really did not notice the change . Thanks for your reviewing. I'll fix it right away.